packages/core/src/feed.ts

```diff
@@ -42,12 +46,12 @@ class DocumentsSubject extends Observable<Document> {
  */
 export interface PublicFeed {
   aggregation: {
-    streamStates: Observable<Document>
+    documents: Observable<Document>
```

Comment:
`documents` its a much better name, thanks for catching that!

Reply:
It is what the spec says :)